### ISSUE TITLE
Bug with integer in LIKE condition

### ISSUE THREAD

**Issue by Drive4ik** [catfan/Medoo#337]
Hello. I found a bug in medoo. Example:
with string value:
```php
$db->debug()->get('table', '*', [
    'field[~]' => '12345'
]);
```
sql result:
```sql
SELECT * FROM "table" WHERE "field" LIKE '%12345%' LIMIT 1
```

but with integer value result is other:
```php
$db->debug()->get('table', '*', [
    'field[~]' => 12345
]);
```
sql result:
```sql
SELECT * FROM "table" WHERE LIMIT 1
```
and 
Warning: Invalid argument supplied for foreach() in ...medoo.php on line 344.
Used latest version 1.0
Please, fix it.

Thank you, @Drive4ik

**Comment by SyuTingSong** [comment]
`LIKE` cause should receive string, not integer.

PR #338 can cheat numeric value as string, so you can use it as auto type casting.

**Comment by catfan** [comment]
This fixed it on 43aca2d8

Thanks to @SyuTingSong request, but in this case it should not be cheated and converted to string value. LIKE condition should accept string value only.

**Comment by SyuTingSong** [comment]
We may receive integer value from some API or whatever, then pass the variable to medoo. Keep `LIKE` accepts string only is okay for me.

@Drive4ik you can pass variable to medoo as `$medoo->select('table', '*', [ 'field[~]' => "$integerValue"])` for type-casting.
